Formula/hex.rb

```diff
@@ -20,6 +20,9 @@ class Hex < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+
+    system "pandoc", "--to=man", "man/hx.1.md", "-o", "hx.1"
```

Comment:
two issues with this
- there is no `man/hx.1.md`
- need to add `pandoc` build dependency